spec/integrity/middleware_order_spec.rb

```diff
@@ -1,49 +1,47 @@
 # frozen_string_literal: true
 
 RSpec.describe "Middleware order" do
-  it "order of middleware in the application is correct" do
-    middlewares = Rails.configuration.middleware.map { |middleware| "#{middleware.inspect}" }
-    expect(middlewares).to eq(
-      %w[
-        BlockRequestsMiddleware
-        TestMultisiteMiddleware
-        ActionDispatch::RemoteIp
-        Middleware::RequestTracker
-        MessageBus::Rack::Middleware
-        ActionDispatch::HostAuthorization
```

Comment:
This middleware isn’t included anymore in the test env. It seems there’s a default configuration when in dev env and that’s why it’s present. But we don’t have any particular configuration for the test env.